server/src/repositories/media.repository.ts

```diff
@@ -92,6 +100,23 @@ export class MediaRepository implements IMediaRepository {
     return pipeline.resize(options.size, options.size, { fit: 'outside', withoutEnlargement: true });
   }
 
+  private getRotationAngleFromOrientation(orientation: ExifOrientation | undefined) {
+    if (!orientation) return {};
+
+    const valueMap: Record<ExifOrientation, { angle?: number, flip?: boolean, flop?: boolean }> = {
```

Comment:
You can make this a constant in `constants.ts` and add `as const`. That would also let you omit the type hint here since TS will know the exact object and its properties.